docs/language-agnostic-definition.md

```diff
@@ -89,10 +97,12 @@ message DefaultMessage{
 }
 ```
 
-- It MUST process `Envelopes` asynchronously.
+- It is recommended that it processes `Envelopes` asynchronously. Note, the specification regarding the processing of messages does not impose any particular implementation choice/constraint; for example, the AEA can process envelopes either synchronously and asynchronously. However, due to the high level of activity that an AEA might be subject to, other AEAs expect a certain minimum level of responsiveness and reactivity of an AEA's implementation, especially in the case of many concurrent dialogues with other peers. That could imply the need for asynchronous programming to make the AEA's implementation scalable.
 
 - It MUST have an identity in the form of, at a minimum, an address derived from a public key and its associated private key.
 
+- It SHOULD implement handling of errors using the `default` protocol.
```

Comment:
Please add the `proto` schema from `default` protocol package here

Reply:
The schema is just above in the document. I mentioned it in text.